lms/djangoapps/bulk_email/models.py

```diff
@@ -4,11 +4,14 @@
 
 
 import logging
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
 
 import markupsafe
 from config_models.models import ConfigurationModel
-from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
```

Comment:
I thought the direct import of the User model was considered bad practice so I updated this while I was updating this file.